### ISSUE TITLE
intermediate results

### ISSUE THREAD

**Issue by aryehw** [yapic/yapic#61]
Is there a way to have yapic generate intermediate results every n epochs?

**Comment by cmohl2013** [comment]
No, currently not. Might be useful, though. To examine training progress directly on images.
